src/test/ReactTestUtils.js

```diff
@@ -376,9 +376,12 @@ ReactShallowRenderer.prototype.getMountedInstance = function() {
   return this._instance ? this._instance._instance : null;
 };
 
+var nextDebugID = 0;
```

Comment:
Can this start at 1? Since this doesn't reset for each shallow renderer it seems prudent to make all of them have non-0 IDs.